app.rb

```diff
@@ -18,9 +18,11 @@ def current_user
 
   get "/" do
     if current_user
-      "You are #{current_user.email}"
+      #"You are #{current_user.email}"
+      erb :index
     else
-      "It works!"
+      #"It works!"
+      erb :other_index
```

Comment:
Can we rename this page to something more descriptive? For example, `sign_in.erb`